### PR TITLE
Fix code block not showing

### PR DESCRIPTION
**Pull request by vpassapera**
Fixes codeblock not showing in the docs.

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -257,9 +257,10 @@ General PHP objects
     models. Additional information is extracted from the PHP doc block comment,
     but the property types must be specified in the JMS annotations.
     
-    NOTE: If you are using serialization contexts (e.g. Groups) each permutation will be treated as a separate Path. For example if you have the following two variations defined in different places in your code:
+    **If you are using serialization contexts (e.g. Groups)**, each permutation will be treated as a separate Path. For example if you have the following two variations defined in different places in your code:
```
Comment (GuilhemN) [inline Resources/doc/index.rst:260]: I don't think this is relevant, this note only applies when you're using the JMS serializer so it shouldn't appear to be some sort of another case in my opinion.

**Review by GuilhemN — COMMENTED** [review]
Good catch, thank you for your PR!

File: Resources/doc/index.rst (hunk 1 above)
Comment (vpassapera) [reply Resources/doc/index.rst:260]: It is indeed relevant to jms serializer contexts as far as I know. I can be explicit about that in the note, so I'll go ahead and update it.

**Comment by GuilhemN** [comment]
I merged https://github.com/nelmio/NelmioApiDocBundle/pull/1851 and https://github.com/nelmio/NelmioApiDocBundle/pull/1846 instead as they are more minimalistic.

Thank you for your PR still!

**Pull request closed without merging**